src/chef_ci_status.rb

```diff
@@ -259,15 +533,61 @@ def print_pr_or_issue_stats(stats, item)
   ) do
     options[:verbose] = true
   end
+
+  opts.on(
+    '--ci-timeout SECONDS',
+    Integer,
+    "Timeout for CI processing in seconds (default: #{options[:ci_timeout]})",
+  ) do |v|
+    options[:ci_timeout] = v
+  end
+
+  opts.on(
+    '--skip-ci',
+    'Skip CI status processing (faster)',
+  ) do
+    # Remove 'ci' from any mode array
```

Comment:
wat. No, this is what `--modes` is for.

Reply:
Yes I think I missed that in the refactor.